anchore_engine/service.py

```diff
@@ -546,6 +546,7 @@ def _init_wsgi_app(self, service_name, api_spec_dir=None, api_spec_file=None):
 
             return self._application
         except Exception as err:
+            logger.exception('Error initializing WSGI application: {}'.format(err))
```

Comment:
`logger.exception()` already captures the traceback and error. It is common to add an error description but not use the `err` variable again since it would be repeated a few lines later.
Since all that is going on, the `traceback.print_exec()` can be removed, and `raise err` can be swapped for `raise`

Reply:
Nice. Great learning moment for me with python logging and error handling.